AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Events/OutboxStatusEvent.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Used as HubPayload for the `OutboxStatus`
+public struct OutboxStatusEvent {
+    /// status of outbox: empty or not
```

Comment:
"`true` if there are no events in the outbox at the time the event was dispatched